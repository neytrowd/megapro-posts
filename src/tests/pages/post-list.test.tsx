import PostList from '../../pages/post-list';
import { screen } from '@testing-library/react';
import { renderWithRouterAndProvider } from '../helpers/render';

describe('Post list', () => {
   test('should contain title', () => {
      renderWithRouterAndProvider(<PostList />);
      expect(screen.getByText(/Post list/i)).toBeInTheDocument();
   });

   test('should match snapshot', async () => {
      renderWithRouterAndProvider(<PostList />);
      expect(screen.getByTestId('post-list-page')).toMatchSnapshot();
   });
});
