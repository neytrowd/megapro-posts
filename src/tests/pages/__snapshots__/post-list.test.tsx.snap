// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post list should been called 1`] = `
<div
  class="MuiBox-root css-0"
  data-testid="post-list-page"
>
  <div
    class="MuiBox-root css-1xn3edc"
  >
    <h4
      class="MuiTypography-root MuiTypography-h4 css-5lbw0b-MuiTypography-root"
    >
      Post list
    </h4>
  </div>
  <div
    class="root MuiBox-root css-0"
  >
    <h6
      class="MuiTypography-root MuiTypography-h6 css-2ulfj5-MuiTypography-root"
    >
      Loading...
    </h6>
  </div>
</div>
`;
